src/Attributes/RequestBody.php

```diff
@@ -22,7 +22,7 @@ public function __construct(
         ?string $request = null,
         ?string $description = null,
         ?bool $required = null,
-        array|JsonContent|XmlContent|Attachable|null $content = null,
+        array|JsonContent|XmlContent|Attachable|MediaType|null $content = null,
```

Comment:
Could you please move `MediaType` further left, right after `array`?
ie: `array|MediaType|JsonContent|XmlContent|Attachable|null $content = null,`

Reply:
The same change needs to be applied to line 16 (phpdoc block). 

Once done the tests will fail since the typehint now does not match the `Annotation\RequestBody::$content` property any more.

So the annotation version of `RequestBody` needs to be updated as well (line 61):
`     * @var array<MediaType>|MediaType|JsonContent|XmlContent|Attachable`